### PR TITLE
Dependencies + configuration for travis + added badge to README

### PR DESCRIPTION
**Pull request by IgnacioColuccio**
### Summary
- Added dependencies and configuration for TravisCI

### Trello card
https://trello.com/c/IcrxfFdT/19-configurar-travisci

### Test plan
See README

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -59,7 +59,10 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
+  adapter: postgresql
   database: igc-rails_test
+  username: postgres
```
Comment (Dinuuu) [inline config/database.yml]: I don't think this is necessary

File: config/database.travis.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+test: &test
+  adapter: postgresql
+  encoding: utf8
+  database: igcrails_test
+  username: postgres
```
Comment (Dinuuu) [inline config/database.travis.yml]: Here you are using the default user postgres, you should use the one you just created "igcrails"

**Comment by IgnacioColuccio** [comment]
Fixed those things

**Comment by Dinuuu** [comment]
:+1:

**Pull request merged**